source/_posts/2016-12-08-policy-files.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Policyfiles"
+date: 2016-12-08 12:21
+comments: true
+categories:
+  - policyfiles
+  - berkshelf
+---
+
+ # Coming Soon!
+
+ This episode will be recorded on Monday, December 19, 21:00 UTC
+
+<!-- more -->
+
+Panel<a name="panel"></a>
+-----
```

Comment:
I typically add folks to this Panel based on the script notes / calendar invite.

https://github.com/foodfight/showz/blob/master/scripts/episode-103-policyfiles.md lists both "Invited" and "Confirmed" panel.  Only confirmed panelists should be listed on the website...and updated as confirmations come in.